src/GrpcProtos/capture.proto

```diff
@@ -139,6 +139,150 @@ message ApiEvent {
   fixed64 r5 = 9;
 }
 
+message ApiScopeStart {
+  // NextID: 16
+
+  int32 pid = 1;
+  int32 tid = 2;
+  uint64 timestamp_ns = 3;
+
+  // We encode the first 64 characters of the scope's name/label using fixed64
+  // values, in order to avoid an additional allocation (for the internal
+  // vector) for short strings. Those allocations have shown a huge performance
+  // impact.
+  // If the name exceeds the 64 characters, the `encoded_name_additional` will
+  // be used to encode the additional characters.
+  fixed64 encoded_name_1 = 4;
+  fixed64 encoded_name_2 = 5;
+  fixed64 encoded_name_3 = 6;
+  fixed64 encoded_name_4 = 7;
+  fixed64 encoded_name_5 = 8;
+  fixed64 encoded_name_6 = 9;
+  fixed64 encoded_name_7 = 10;
+  fixed64 encoded_name_8 = 11;
+  bytes encoded_name_additional = 12;
+
+  // "rgba" syntax.
```

Comment:
So alpha the lowest bits? That's the one that was used in `EncodedEvent`, right?

Reply:
Exactly. That one is also the one described in Orbit.h. In particular it is the enum "orbit_api_color".

Reply:
Thinking about it, I would just rename the field to `color_rgba`.

Reply:
I always find it confusing whether a color it's ARGB or RGBA so I think it's clearer to always have it in the field.

Reply:
For example I think that Qt uses ARGB.

Reply:
Done.